teps/tools/tep-template.md.template

```diff
@@ -127,17 +132,18 @@ and make progress.
 Describe the concrete improvement specific groups of users will see if the
 Motivations in this doc result in a fix or feature.
 
-Consider both the user's role (are they a Task author? Catalog Task user?
-Cluster Admin? etc...) and experience (what workflows or actions are enhanced
-if this problem is solved?).
+Consider the user's:
+- role - are they a Task author? Catalog Task user? Cluster Admin? e.t.c.
```

Comment:
maybe link to our user profiles: https://github.com/tektoncd/community/blob/main/user-profiles.md

Reply:
added, thank you!